src/tracer/tracer.ts

```diff
@@ -182,13 +187,59 @@ export class AutoblocksTracer {
     return data.traceId;
   }
 
+  private async sendTestEventUnsafe(
+    message: string,
+    args?: SendEventArgs,
+  ): Promise<string> {
+    const cliServerAddress = readEnv(
+      AutoblocksEnvVar.AUTOBLOCKS_CLI_SERVER_ADDRESS,
+    );
+    if (!cliServerAddress) {
+      throw new Error('Tried to send test event without a CLI server address.');
+    }
+    const store = testCaseRunAsyncLocalStorage.getStore();
+    if (!store) {
+      throw new Error('Tried to send test event outside of test run.');
+    }
+    const traceId = args?.traceId || this.traceId || crypto.randomUUID();
```

Comment:
the return type of `sendEvent` is `Promise<{ traceId?: string }>` so we don't really need to generate a traceId just for the sake of returning it. we can just return `{}`

then we can keep this part inside `mergeProperties` instead of repeating it:

```ts
const traceId = args?.traceId || this.traceId;
const timestamp = args?.timestamp || new Date().toISOString();
```

and then something like:

```ts
const {
  traceId,
  timestamp,
  properties,
} = this.prepareRequest(args);
```

wdyt?

Reply:
oh ya that makes sense

Reply:
Thinking about this more, i'm just gonna make the breaking change to not return traceId from either here. i'll keep it as object though. I think it's weird if both behave differently in test vs prod context because if you do rely on it prod context your traces wont' show correctly in test context when we add trace view there

Reply:
i would guess no one has actually done anything w/ the response yet so i'd just make it `Promise<void>`